cl/assets/tailwind/input.css

```diff
@@ -11,9 +11,11 @@
     --mobile-main-padding: theme(spacing.4);
     --padded-full-width: calc(100vw - (2 * var(--mobile-main-padding)));
     --corpus-search-height: theme(spacing.12);
+    --desktop-header-height: theme(spacing.26);
+    --desktop-header-filters-height: calc(100svh - var(--desktop-header-height));
     --mobile-dialog-header-height: theme(spacing.18);
     --mobile-dialog-panel-height: calc(100vh - var(--mobile-dialog-header-height));
-    --content-max-width: 1400px;
+    --content-max-width: 1200px;
```

Comment:
The tweak seems to have impacted the search component. Specifically, I've noticed that the date fields are smaller now, which causes the date to be cropped when the calendar component is used. Here's a screenshot for your reference:

<img width="2632" height="1436" alt="image" src="https://github.com/user-attachments/assets/ba274c98-98b1-4401-9577-f41a3e4e5b54" />